src/components/LoadingShape/styled/index.js

```diff
@@ -0,0 +1,119 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import AvatarIcon from '../icons/avatar';
+import ImageIcon from '../icons/image';
+
+const StyledShapeContainer = styled.div`
+    position: relative;
+    width: 100%;
+    height: 100%;
+`;
+
+const StyledImageIcon = styled(ImageIcon)`
+    ${props =>
+        props.shape === 'square' &&
+        `
+        width: 85%;
+        `}
+
+    ${props =>
+        props.shape === 'circle' &&
+        `
+        width: 60%;
+        `}
+
+    ${props =>
+        (props.shape === 'rect' || props.shape === 'rounded-rect') &&
+        `
+        max-width: 25%;
+        height: 80%;
+        `}
+`;
+
+const StyledAvatarIcon = styled(AvatarIcon)`
+    ${props =>
+        props.shape === 'square' &&
+        `
+        width: 85%;
+        `}
+
+    ${props =>
+        props.shape === 'circle' &&
+        `
+        width: 60%;
+        `}
+
+    ${props =>
+        (props.shape === 'rect' || props.shape === 'rounded-rect') &&
+        `
+        max-width: 25%;
+        height: 80%;
+        `}
+`;
+
+const StyledLoadingShape = attachThemeAttrs(styled.div)`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    background: ${props => props.palette.background.secondary}
+        linear-gradient(
+            90deg,
+            rgba(226, 228, 233, 0.01) 0%,
```

Comment:
let's use the color calculated on the theme,
 on the Table component, we made something like that
```
        linear-gradient(
            90deg,
            ${props => replaceAlpha(props.palette.background.highlight, 0.01)} 0%,
            ${props => replaceAlpha(props.palette.background.highlight, 0.65)} 50%,
            ${props => replaceAlpha(props.palette.background.highlight, 0.01)} 100%
        );
```